src/pages/ExamsPage/data/api.js

```diff
@@ -0,0 +1,16 @@
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+function getExamsBaseUrl() {
+  ensureConfig([
+    'EXAMS_BASE_URL',
+  ]);
+  return getConfig().EXAMS_BASE_URL;
+}
+
+/* eslint-disable import/prefer-default-export */
+export async function getCourseExams(courseId) {
+  const url = `${getExamsBaseUrl()}/api/v1/exams/course_id/${courseId}/`;
```

Comment:
Question: is there no call to legacy/proctoring needed here? I'm guessing because exams dashboard only works with exams, so there is no accounting for legacy experience before deprecating here?

Reply:
yup only edx-exams, we don't need to deal with edx-proctoring at all 🎉